filebeat/scripts/jenkins/unit-test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+source ./dev-tools/common.bash
```

Comment:
This file isn't in root folder so I guess that it's called from there or it won't find `./dev-tools`

Reply:
Yes, it's called from the root folder. See https://github.com/elastic/beats/pull/9778/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R90

Reply:
Awesome, so you are good to go :)